integration/e2e/e2e_test.go

```diff
@@ -213,7 +220,13 @@ var _ = Describe("EndToEnd", func() {
 		BeforeEach(func() {
 			network = nwo.New(nwo.BasicKafka(), testDir, client, StartPort(), components)
 			network.MetricsProvider = "prometheus"
+
 			network.ChannelParticipationEnabled = true
+			fileRepoDir := filepath.Join(testDir, "filerepo")
+			network.ChannelParticipationFileRepoDir = fileRepoDir
+			err := os.MkdirAll(fileRepoDir, 0755)
+			Expect(err).NotTo(HaveOccurred())
+
```

Comment:
So, does this mean we've replaced our `non-participation` tests with ones that enable it? Where are the legacy tests?

Reply:
These tests enable the channel participation API, but given that the system channel exists, it can only be used for listing channels. These tests still join using "legacy" functionality and simply verify that listing channels is possible.